### ISSUE TITLE
Package request : Focalboard (from Mattermost)

### ISSUE THREAD

**Issue by gfauredev** [NixOS/nixpkgs#190459]
**Project description**

Package Focalboard, a modern project and task management app.

**Metadata**

* homepage URL: https://www.focalboard.com/
* source URL: https://github.com/mattermost/focalboard
* license: MIT
* platforms: Linux, macOS

**Comment by felixsanz** [comment]
@gfauredev did you manage to package it?

**Comment by gfauredev** [comment]
No, it looks like I overestimated my interest in packaging focalboard, as I moved to other tools now. I'm sorry if this disincentivised people to package it before.
